### ISSUE TITLE
Site visit not being populate when retrieving from deployment. 

### ISSUE THREAD

**Issue by Elijahwalkerwest** [UCHIC/ODM2Sensor#270]
When retrieving from deployment details, it doesn't auto-populate the site visit. This is problematic. 

Have solution, haven't pushed yet.

**Comment by Elijahwalkerwest** [comment]
Deployed.

**Comment by AmberSJones** [comment]
I actually don't think we want it to auto-populate the site visit. If it does, it populates with the same site visit for which the deployment occurred - which is not likely to be when the retrieval occurred. What would be helpful, though, is to ensure that the site visit selected for retrieval has a DateTime that is AFTER the DateTime of the site visit associated with the deployment.

**Comment by Elijahwalkerwest** [comment]
Sounds good, changing this behavior, will push soon.

**Comment by AmberSJones** [comment]
Ok - I think this is ok, though there is no logic to ensure that the retrieval occurs after the deployment. How hard would that be to implement?

**Comment by Elijahwalkerwest** [comment]
Let me look in to it, and I will try and work this in with the next batch of merges.

**Comment by AmberSJones** [comment]
This is unchanged, right?

**Comment by Elijahwalkerwest** [comment]
Yes, sorry.

**Comment by Elijahwalkerwest** [comment]
Logic to ensure retrieval occurs after deployment now exists.

**Comment by AmberSJones** [comment]
I don't think this is performing as desired. It appears that the logic is there because when I select a site visit for retrieval that was prior to the deployment, I get an error, while if I select a site visit visit for retrieval that is after the deployment, it correctly retrieves. However, this behavior is not helpful or clear to the user. My preference is that the site visits prior to the deployment are not even options in the drop down. Another potential solution is that the user is returned to the form with a message that states "Select a Site Visit for Retrieval that is after the Site Visit of the Deployment."

**Comment by Elijahwalkerwest** [comment]
I see! I will get on that right now.
